src/main/java/org/opensearch/security/rest/TenantInfoAction.java

```diff
@@ -93,9 +92,13 @@ public TenantInfoAction(final Settings settings, final RestController controller
         this.configurationRepository = configurationRepository;
     }
 
-    @Override
     public List<Route> routes() {
-        return routes;
+        return ImmutableList.of();
```

Comment:
same

Reply:
My bad, they should be deleted as well, fixed